brody_lab_to_nwb/interfaces/protocol_info/ksphysortingextractor.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+import spikeextractors as se
+
+
+class ksphySortingExtractor(se.SortingExtractor):
```

Comment:
@CodyCBakerPhD this looks much more general than a KS-Phy extractor. Why not calling it something like `RuntimeSortingExtractor` or `CustomSortingExtractor`? I think that it has nothing specific to kilosort and phy

Reply:
Since this is in the Brody repo, I'm not overly concerned about the name. However, as @CodyCBakerPhD and I discussed, this name suggests that they are using phy, and therefore might have an upstream Phy file which we already support in NCT, so it's worth asking to see if we can access those files instead.

Reply:
I have access to the files now, but there's indications the data in these .mat files may be additionally processed from that. I'll still add the PhySortingInterface to the SpikeGadgets converter though, in case they ever want to convert directly from that.

From my conversations with Jess though, both of these approaches to saving the sorting data to NWB are only for historical data generated through these methods; they fully intend to switch the SpikeGadgets stuff to processing via SpikeInterface once we help them get that setup.